### DISCUSSION TITLE
Upgrading Argo Rollouts on cluster

### DISCUSSION THREAD

**Discussion by daniel-hirvonen** [argoproj/argo-rollouts#2308] [Q&A] [answered]
Hi
Since there is only clear instructions about installing Argo rollouts, how is it when it comes to upgrades and potential downtimes etc during those parts?

Will there be any downtime during a new apply / replace on the latest version during an upgrade?

**Comment by zachaller** [comment, marked as answer]
There should not be downtime any current reconciliations would pause and continue when the new controller comes online.
